textpipe/doc.py

```diff
@@ -14,7 +14,7 @@
 from bs4 import BeautifulSoup
 
 
-class TextpipeMissingModelException(Exception):
+class MissingModelException(Exception):
```

Comment:
This rename breaks backwards compatibility. Don't think we need it? If we do, we should bump the version to 7.

Reply:
Fixed (reverted rename)